packages/amplify-graphql-api-construct/API.md

```diff
@@ -36,27 +36,27 @@ export type AmplifyGraphqlApiProps = {
     schema: AmplifyGraphqlApiSchema;
     apiName?: string;
     authorizationConfig: AuthorizationConfig;
-    referencedFunctions?: Record<string, IFunction>;
+    functionNameMap?: Record<string, IFunction>;
     conflictResolution?: ConflictResolution;
     stackMappings?: Record<string, string>;
```

Comment:
maybe this should be `stackNameMap` to be consistent with `functionNameMap`?

Reply:
Discussed, and leaving this as `stackMappings` today to reflect what this is in the Amplify CLI realm. https://docs.amplify.aws/cli/graphql/override/#place-appsync-resolvers-in-custom-named-stacks